sensitivity.tex

```diff
@@ -0,0 +1,126 @@
+\section{Sensitivity Analysis}
+
+The most important aspect of any fuel cycle transition scenario
+is the accrual of fissile materials for new reactor deployment.
+The collaborative strategy makes a transition possible materialistically,
+but the aggressive transition demands an increase in reprocessing capacity.
+This is due to both high initial plutonium loading of the \gls{ASTRID}
+reactors and the low plutonium composition in spent \gls{UOX} fuel.
+
+We varied two parameters, the lifetime of French \glspl{LWR} and the
+breeding ratio of \gls{ASTRID} reactors. The parameters and their values
+are listed in \cref{tab:sen_par}
+
+\begin{table}[h]
+    \centering
+    \begin{tabularx}{\textwidth}{lbb}
+        \hline
+        \textbf{Parameter} & \textbf{Original} & \textbf{Values} \\
+        \hline
+        Lifetime of French \glspl{LWR} [years] & 60  & 65, 70, 80 \\
+        Breeding Ratio of \glspl{ASTRID} & 1.08 & 1.11, 1.15, 1.18 \\ 
+        \hline
+    \end{tabularx}
+    \caption {Two parameters are varied for the sensitivity analysis - the 
+              lifetime of French \glspl{LWR} and the breeding ratio of \glspl{ASTRID}.
+              The parameters are increased to analyze their impact on the reprocessing
+              demand for the transition.}
+    \label{tab:sen_par}
+\end{table}
+
+\subsection{Breeding Ratio}
+
+The result of the breeding ratio sensitivity analysis is shown in
+\cref{fig:br_rep}.
+Increase in the breeding ratio of \gls{ASTRID} reactors
+decreases the monthly average reprocessing, but the 
+average \gls{UOX} reprocessing demand doesn't dramatically change,
+because the time that \gls{ASTRID} fuel supplied from reprocessed
+\gls{UOX} decrease with increase breeding ratio. The average monthly reprocessing
+demands are calculated by dividing the total amount by the number of non-zero elements.
+
+\begin{figure}[htbp!]
+    \begin{center}
+        \includegraphics[scale=0.6]{./images/sensitivity/br.png}
+    \end{center}
+    \caption{Increasing the breeding ratio decreases the average monthly reprocessing for all fuels,
+             but the average monthly \gls{UOX} reprocessing does not decrease due to shorter supply time. }
```

Comment:
not sure what you mean by "supply time"

Reply:
I was trying to say that the increase in breeding ratio leads to the SFR being 'independent' quicker, meaning that the reprocessed UOX will be not-needed quicker.

In other words,
average reprocessing demand =
 `Total Reprocessing Demand for UOX` / `Months of Reprocessing UOX`

Since `Months of Reprocessing UOX` decrease with the increase of Breeding Ratio, the average does not change much.